### PR TITLE
Added social healthcare information response

### PR DESCRIPTION
**Pull request by ahamdany**
Description
----
Added social healthcare information support for coverage.

Screen Shots
----
Before

Screen Shot 2020-09-24 at 9 07 58 AM

Screen Shot 2020-09-24 at 1 06 18 PM

After

Screen Shot 2020-09-23 at 2 11 10 PM

Screen Shot 2020-09-24 at 2 49 14 PM

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/financial/coverage.md [hunk 1]
```diff
@@ -89,6 +89,14 @@ Search for Patient-level or Encounter-level Coverages that meet supplied query p
 <%= headers status: 200 %>
 <%= json(:r4_coverage_encounter_bundle) %>
 
+
+#### Response when social healthcare information is returned
```
Comment (sydneanschutz) [inline content/millennium/r4/financial/coverage.md:93]: are we calling this social or public healthcare externally?

Comment (ahamdany) [reply content/millennium/r4/financial/coverage.md]: Updated [here](https://github.com/cerner/fhir.cerner.com/commit/bfdeb4b16e342fb06798c33c70555aec4b30329e#diff-81b4004b2ae7fd9a288412597630810cL93)

**Comment by ahamdany** [comment]
Added implementation notes [here](https://github.com/cerner/fhir.cerner.com/commit/bfdeb4b16e342fb06798c33c70555aec4b30329e#diff-81b4004b2ae7fd9a288412597630810cR53)

File: content/millennium/r4/financial/coverage.md [hunk 2]
```diff
@@ -49,6 +49,10 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
 Search for Patient-level or Encounter-level Coverages that meet supplied query parameters:
 
     GET /Coverages?:parameters
+    
+_Implementation Notes_
+  
+* Public Healthcare (PH) represents an insurance policy funded by a public health system such as a provincial or national health plan.
```
[the remark this answers is not in the document]
Comment (ahamdany) [reply content/millennium/r4/financial/coverage.md:55]: I thought having the abbreviation in the description was enough, but do suggest if it needs to be in some other format.

[the remark this answers is not in the document]
Comment (ahamdany) [reply content/millennium/r4/financial/coverage.md:55]: Alex and Madhur suggested that we have this.

**Comment by ahamdany** [comment]
Removed example [here](https://github.com/cerner/fhir.cerner.com/pull/469/commits/748da620231f21c4b2f4cd7eee37454f09b86146)

**Review by sydneanschutz — APPROVED** [review]

File: content/millennium/r4/financial/coverage.md [hunk 3]
```diff
@@ -89,6 +93,11 @@ Search for Patient-level or Encounter-level Coverages that meet supplied query p
 <%= headers status: 200 %>
 <%= json(:r4_coverage_encounter_bundle) %>
 
+#### Response when public healthcare information is returned
```
Comment (MadhurRaj) [inline content/millennium/r4/financial/coverage.md:96]: let's put this in its own example.

Example - Encounter-level Public Healthcare Coverage

**Review by MadhurRaj — CHANGES_REQUESTED** [review]

File: content/millennium/r4/financial/coverage.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ahamdany) [reply content/millennium/r4/financial/coverage.md]: Updated again [here](https://github.com/cerner/fhir.cerner.com/pull/469/commits/417efdfee05ede5125e471fdd7e74edfb13c7e35#diff-81b4004b2ae7fd9a288412597630810cL55)

File: content/millennium/r4/financial/coverage.md (hunk 3 above)
Comment (ahamdany) [reply content/millennium/r4/financial/coverage.md]: added example [here](https://github.com/cerner/fhir.cerner.com/pull/469/commits/60a49b0cd9a1736f87d9db02aec450bd377466b3)

File: content/millennium/r4/financial/coverage.md [hunk 4]
```diff
@@ -93,12 +93,14 @@ _Implementation Notes_
 <%= headers status: 200 %>
 <%= json(:r4_coverage_encounter_bundle) %>
 
+#### Example - Encounter-level Public Healthcare Coverage
+
 #### Request
 
     GET https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Coverage?-encounter=97701467
```
Comment (ahamdany) [inline content/millennium/r4/financial/coverage.md]: removed empty line [here](https://github.com/cerner/fhir.cerner.com/pull/469/commits/ef93c1d3bdf0cf2bdd76d2924ee83b060ec7c133#diff-81b4004b2ae7fd9a288412597630810cL102)

**Review by exopsRamba — APPROVED** [review]

**Review by MadhurRaj — APPROVED** [review]

**Comment by lalitha94** [comment]
Verified on live
Screen Shot 2020-09-24 at 6 14 24 PM
Screen Shot 2020-09-24 at 6 15 54 PM

**Pull request merged**